HungerGamesSimulator/Data/Actor.cs

```diff
@@ -18,15 +24,32 @@ protected Actor( string name )
     Name = name;
   }
 
-  public virtual ActorStates GetState()
+  public virtual ActorStates GetNewState()
   {
     if ( Health <= 0 )
     {
       return ActorStates.Dead;
     }
 
-    var coin = Random.Shared.Next( 0, 2 );
-    return coin == 0 ? ActorStates.Attacking : ActorStates.Moving;
+    if ( IsInParty() )
+    {
+      if ( SimulationUtils.RollD20() < stayInPartyDC )
+      {
+        return ActorStates.LeaveParty;
```

Comment:
idk if this matters but it might be good to have players when leaving a party, either triggering a move or a fight? or would it just strigger a fight the next day? But then pegs the question, why wait to fight for a whole day if theyre disbanding that day?